libr/include/r_util/r_str.h

```diff
@@ -150,6 +150,7 @@ R_API char *r_str_path_escape(const char *path);
 R_API int r_str_unescape(char *buf);
 R_API char *r_str_escape(const char *buf);
 R_API char *r_str_escape_sh(const char *buf);
+R_API char *r_str_escape_r2(const char *buf);
```

Comment:
this is out of scope for this PR and considering the amount of r2-shell injections we have, I don't think this single escape will "save the world". I would much rather have this in a separate PR with *a lot* of unit tests to make sure it works as intended.

Reply:
its needed for pxr*, its not really escaping, just filtering dangerous chars. i can do a separate PR for this indeed